lib/loadGltfUris.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+var fs = require('fs');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+var dataUriToBuffer = require('data-uri-to-buffer');
+
+module.exports = loadGltfUris;
+
+function loadGltfUris(path, gltf, callback) {
+    var shaders = gltf.shaders;
+    if (defined(shaders)) {
+        for (var id in shaders) {
+            if (shaders.hasOwnProperty(id)) {
+                var shader = shaders[id];
+                shader.extras = shader.extras || {};
+
+                var uri = shader.uri;
+                loadURI(gltf, path, shader.extras, uri, callback);
+            }
+        }
+    }
+
+    return gltf;
+}
+
+function loadURI(gltf, path, extras, uri, callback) {
+    if (!/^data\:/i.test(uri)) {
```

Comment:
Create a separate function in a separate file (and separate test) for `isDataUri` since we will use this in a few places.